### ISSUE TITLE
Shiny app cannot generate data with error 'could not find function "ga_ts"'

### ISSUE THREAD

**Issue by wkrause13** [ykang/tsgeneration#35]
I am attempting to use the shiny app launched by calling app_gratis(), however, there are a number of functions called in the shiny app that are no longer exported.

I was able to get the app working by manually exporting or recreating the missing functions. 

```R
install.packages("doParallel")
eval(parse(text=paste0('ga_ts', '<-gratis:::', 'ga_ts')))
eval(parse(text=paste0('fitness_ts', '<-gratis:::', 'fitness_ts')))
nsdiffs1 <- function(x) {
  c(nsdiffs = ifelse(frequency(x) == 1L, -1, forecast::nsdiffs(x)))
}
```

I am not sure if that's the best approach, but it looks sufficient to do some exploratory work.

**Comment by robjhyndman** [comment]
Sorry about that. When I rewrote a lot of the package, I didn't update the Shiny app. Feel free to do a PR with the required fixes.

**Comment by feng-li** [comment]
It was solved with version 1.0.1.
